src/apps/basic-transaction/index.tsx

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import styled from 'styled-components';
+
+// components
+import useBottomMenuModal from '../../hooks/useBottomMenuModal';
+import TextInput from '../../components/Form/TextInput';
+import Button from '../../components/Button';
+import FormGroup from '../../components/Form/FormGroup';
+import Label from '../../components/Form/Label';
+import Select from '../../components/Form/Select';
+
+// utils
+import { visibleChains } from '../../utils/blockchain';
+
+const App = () => {
+  const [t] = useTranslation();
+  const [destinationAddress, setDestinationAddress] = React.useState<string>('');
+  const [value, setValue] = React.useState<string>('');
+  const [data, setData] = React.useState<string>('');
+
+  const chainOptions = visibleChains.map((chain) => ({
+    id: `${chain.id}`,
+    title: chain.name,
+    value: chain.id,
+  }));
+
+  const [chainId, setChainId] = React.useState<number>(+chainOptions[0]?.value);
+  const { show } = useBottomMenuModal();
+
+  const send = () => {
+    if (!destinationAddress) return;
+    show({
+      type: 'send',
+      payload: {
+        title: 'Basic Transaction',
+        subtitle: `This will execute basic transaction to ${destinationAddress} address with value ${value} and ${data ? 'attached' : 'no'} call data on ${chainId} chain`,
```

Comment:
I think `description` is nicer here as mentioned on Slack

Reply:
Changed!